File: eAirFlow.Model/Models/Employee.cs
using System;
using System.Collections.Generic;

namespace eAirFlow.Model.Models
{

    public class Employee
    {
        public int EmployeeId { get; set; }
        public int? UserId { get; set; }

        public string? Name { get; set; }

        public string? Surname { get; set; }

        public string? Email { get; set; }

        public string? PhoneNumber { get; set; }

        public DateTime? HireDate { get; set; }

        public Position? Position { get; set; }
        public int? PositionId { get; set; }
        public int? AirportId { get; set; }
        public Airport? Airport { get; set; }


    }
}

File: UI/eairflow_mobile/lib/models/employee.dart
import 'package:eairflow_mobile/models/airport.dart';

import 'position.dart';

class Employee {
  int? employeeId;
  int? userId;
  int? positionId;
  Position? position;
  Airport? airport;

  Employee();

  Employee.fromJson(Map<String, dynamic> json) {
    employeeId = json['employeeId'];
    userId = json['userId'];
    positionId = json['positionId'];
    position = json['position'] != null
        ? Position.fromJson(json['position'])
        : null;
    airport = json['airport'] != null
        ? Airport.fromJson(json['airport'])
        : null;
  }
}
